src/App.jsx

```diff
@@ -0,0 +1,194 @@
+import { useEffect, useState } from 'react';
+import S from './App.module.css';
+import { PandaFace } from './images';
+import { getProductList } from './utils/Getproduct';
+import { Pagination } from './components/Pagination';
+import { favoriteCount } from './utils/FavoriteCount';
+
+function App() {
+  const [bestItems, setBestItems] = useState([]);
+  const [allItems, setAllItems] = useState([]);
+  const [orderBy, setOrderBy] = useState('recent');
+  const [page, setPage] = useState(1);
+  const [searchTerm, setSearchTerm] = useState('');
+  const [filteredItems, setFilteredItems] = useState([]);
+
+  const fetchBestList = async () => {
+    const { list } = await getProductList({
+      page: 1,
+      pageSize: 4,
+      orderBy: 'favorite',
+    });
+
+    setBestItems(list);
+  };
+
+  const handleBestFavoriteCount = async id => {
+    const favorite = await favoriteCount(id);
+    console.log(favorite);
+    if ('message' in favorite) {
+    }
+    fetchBestList();
+  };
+
+  useEffect(() => {
+    fetchBestList();
+  }, []);
+
+  const fetchAllList = async () => {
+    const { list } = await getProductList({
+      page,
+      pageSize: 10,
+      orderBy,
+    });
+
+    setAllItems(list);
+  };
+
+  // 상품 좋아요 수 상태 관리
+  const handleAllFavoriteCount = async id => {
+    const favorite = await fetchAllList(id);
+
+    if ('message' in favorite) {
+      console.log('좋아요 안눌림...');
+    }
+    fetchAllList();
+  };
+
+  useEffect(() => {
+    fetchAllList();
+  }, [page, orderBy]);
+
+  // 상품 검색창 상태 관리
+  useEffect(() => {
+    if (searchTerm === '') {
+      setFilteredItems(allItems);
+    } else {
+      const filtered = allItems.filter(item =>
+        item.description.toLowerCase().includes(searchTerm.toLowerCase())
+      );
+      setFilteredItems(filtered);
+    }
+  }, [searchTerm, allItems]);
+
+  const handleSearchChange = e => {
+    setSearchTerm(e.target.value);
+  };
+
+  return (
+    <div>
+      <header>
+        <div className={S.header}>
+          <div className={S.header_left}>
+            <img
+              className={S.header_logo}
+              src={PandaFace}
+            ></img>
```

Comment:
```suggestion
            <img
              className={S.header_logo}
              src={PandaFace}
              alt="PandaLogo"
             />
```
children이 없는 태그는 이렇게 많이 작성합니다!
그리고 alt도 추가해주세요~~

Reply:
[f1875ed](https://github.com/codeit-bootcamp-frontend/7-Sprint-Mission/pull/185/commits/f1875edc1e3bc56c652b94680fb574dbbdd3b2be) ✅